src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
```

Comment:
Add a space after 'My name is' to properly render the name.

Reply:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/70220667/237016532-dc703d6f-44bb-4e5f-8c26-b866fa87df68.png">